app/src/main/java/de/westnordost/streetcomplete/quests/postbox_royal_cypher/PostboxRoyalCypher.kt

```diff
@@ -0,0 +1,12 @@
+package de.westnordost.streetcomplete.quests.postbox_royal_cypher
+
+enum class PostboxRoyalCypher(val osmValue: String, val wikidataValue: String) {
+    VICTORIA("VR", "Q33102356"),
+    EDWARDVII("EVIIR", "Q33102324"),
+    GEORGEV("GR", "Q33102273"),
+    EDWARDVIII("EVIIIR", "Q33102195"),
+    GEORGEVI("GVIR", "Q33102169"),
+    ELIZABETHII("EIIR","Q33102113"),
+    SCOTTISHCROWN("scottish_crown", ""),
```

Comment:
Why tagging the wikidata at all? Isn't or shouldn't the wikidata be assigned to that OSM tag already? That seems like duplication.

Reply:
I guess so, it was there on the wiki so I assumed it was something that might be helpful to have tagged.
Surely something similar could be said about a lot of wikidata tags (e.g. for brands), but then I don't really know what uses the wikidata tags.

Reply:
Ok then lets remove the wikidata tagging.

Reply:
Have done